ferrischat_webserver/src/channels/delete_channel.rs

```diff
@@ -23,7 +22,6 @@ pub async fn delete_channel(req: HttpRequest, _: crate::Authorization) -> impl R
         Ok(resp) => match resp {
             Some(channel) => Channel {
                 id: bigdecimal_to_u128!(channel.id),
-                guild_id: bigdecimal_to_u128!(channel.guild_id),
```

Comment:
Readd this line, as it was required to initialize the struct. It'll still work since you're fetching the data from the database not the URL.

Reply:
Will fix soon

Reply:
Resolved